### PR TITLE
feat: added payload and end to end tests for multichain execution

### PR DESCRIPTION
**Pull request by sendra**

### CODE REVIEW

**Review by sakulstra — APPROVED** [review]

File: script/DeployPolygonFrax.s.sol [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+* to deploy copy .env.example to .env and fill it, and then execute:
+* source .env
+* forge script ./script/DeployPolygonFrax.s.sol --fork-url $RPC_URL --private-key $PRIVATE_KEY --verify --etherscan-api-key $ETHERSCAN_API_KEY -vvvv
+*/
+contract DeployPolygonFrax is Script {
+  function run() external {
+    vm.startBroadcast();
+    new FraxPayload();
```
Comment (eboadom) [inline script/DeployPolygonFrax.s.sol:15]: Probably a good idea to assign to a variable and log it

File: src/contracts/polygon/FraxPayload.sol [hunk 2]
```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Polygon} from 'aave-address-book/AaveV3Polygon.sol';
+import {IPoolConfigurator, ConfiguratorInputTypes} from 'aave-address-book/AaveV3.sol';
+import {IERC20Metadata} from 'solidity-utils/contracts/oz-common/interfaces/IERC20Metadata.sol';
+import {IProposalGenericExecutor} from '../../interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @dev This steward enables FRAX as collateral on AAVE V3 Polygon
```
Comment (eboadom) [inline src/contracts/polygon/FraxPayload.sol:10]: This stewards lists FRAX as collateral and borrowing asset on Aave V3 Polygon

File: src/contracts/polygon/FraxPayload.sol [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  // New asset being listed (FRAX)
+  // **************************
+
+  address public constant UNDERLYING =
+    0x45c32fA6DF82ead1e2EF74d17b76547EDdFaFF89;
+  string public constant ATOKEN_NAME = 'Aave Polygon FRAX';
+  string public constant ATOKEN_SYMBOL = 'aPolFRAX';
+  string public constant VDTOKEN_NAME = 'Aave Polygon Variable Debt FRAX';
+  string public constant VDTOKEN_SYMBOL = 'variableDebtPolFRAX';
+  string public constant SDTOKEN_NAME = 'Aave Fantom Stable Debt FRAX';
```
Comment (eboadom) [inline src/contracts/polygon/FraxPayload.sol:35]: Not Fantom

File: src/test/PolygonFraxE2E.t.sol [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+      vm,
+      FRAX_WHALE,
+      FRAX_WHALE,
+      FRAX,
+      200 ether,
+      2,
+      vFRAX
+    );
+
+    // TODO: uncomment when foundry fixes bug with public methods on libs
```
Comment (eboadom) [inline src/test/PolygonFraxE2E.t.sol:195]: Any update on this?

Comment (sendra) [reply src/test/PolygonFraxE2E.t.sol:195]: it still not solved looks like. this is the issue that mentions it: https://github.com/foundry-rs/foundry/issues/2549

**Review by eboadom — APPROVED** [review]

**Pull request merged**